src/aat/java/uk/gov/hmcts/reform/em/stitching/functional/DocumentTaskScenarios.java

```diff
@@ -137,4 +138,26 @@ public void testStitchDocumentsWithSortIndices() throws IOException, Interrupted
         Assert.assertTrue(indexOfDocument2 < indexOfDocument1);
     }
 
+
+    @Test
+    public void testPostBundleStitchWithCallback() throws IOException {
+        BundleDTO bundle = testUtil.getTestBundle();
+        DocumentTaskDTO documentTask = new DocumentTaskDTO();
+        documentTask.setBundle(bundle);
+
+        CallbackDto callback = new CallbackDto();
+        callback.setCallbackUrl("http://some-callback.fak");
+
+        documentTask.setCallback(callback);
+
+        Response createTaskResponse = testUtil.authRequest()
+                .log().all()
```

Comment:
Can we take the logging out

Reply:
it's tests - we should see the logs

Reply:
when we need to debug a failing test, otherwise it's just a stream of stuff in the jenkins output that makes it difficult to work out what's going on

Reply:
do you suggest to turn it on when a test fails? how?

Reply:
I would just do it manually as part of investigating why the test is failing

Reply:
I don't fully agree with you point about polluting Jenkins output. One looks in there to find issues when a build fails. It's better to have all information at hand when it happens and it's not that difficult to distinguish between different stages of deployment.